css/index.css

```diff
@@ -1 +1,565 @@
+}
+
+.services-container .main-content .ux-ui {
+    display: flex;
+    flex-wrap: wrap;
+    border: 1px solid black;
+    height: 275px;
+    width: 375px;
+    margin: 20px;
+    padding: 20px;
+    background-color: #f5f5f5;
+}
+
+.services-container .main-content .digital-marketing {
+    display: flex;
+    flex-wrap: wrap;
+    border: 1px solid black;
+    height: 275px;
+    width: 375px;
+    margin: 20px;
+    padding: 20px;
+    background-color: #f5f5f5;
+}
+
+.services-container .main-content .web-development {
+    display: flex;
+    flex-wrap: wrap;
+    border: 1px solid black;
+    height: 275px;
+    width: 375px;
+    margin: 20px;
+    padding: 20px;
+    background-color: #f5f5f5;
+}
+
+.services-container .main-content .ios-development {
+    display: flex;
+    flex-wrap: wrap;
+    border: 1px solid black;
+    height: 275px;
+    width: 375px;
+    margin: 20px;
+    padding: 20px;
+    background-color: #f5f5f5;
+}
+
+.services-container .main-content .android-development {
+    display: flex;
+    flex-wrap: wrap;
+    border: 1px solid black;
+    height: 275px;
+    width: 375px;
+    margin: 20px;
+    padding: 20px;
+    background-color: #f5f5f5;
+}
+
+.bottom-section {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    padding-top: 35px;
+    padding-left: 20px;
+    height: 400px;
+}
+
+.bottom-section h2 {
+    width: 375px;
+    height: 20px;
+}
+
+.bottom-section p {
+    width: 375px;
+}
+
+.bottom-section ol {
+    width: 375px;
+    height: 100px;
+    list-style-type: square;
+    list-style-position: inside;
+    line-height: 1.2;
+    margin-top: 20px;
+}
+
+.bottom-section .services-info-img {
+    width: 375px;
+    height: auto;
+}
+
+@media (max-width: 800px) {
+    .container {
+        width: 100%;
+        margin: 0 auto;
+    }
+
+    .main-head {
+        flex-direction: column-reverse;
+        justify-content: center;
+        align-items: center;
+    }
+
+    .main-head nav {
+        display: flex;
+        justify-content: space-between;
+    }
+
+    .cta {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 38.62%;
+    }
+
+    .cta img {
+        display: none;
+    }
+
+    .cta-text h1 {
+        font-size: 7rem;
+    }
+
+    .middle-img {
+        display: flex;
+        justify-content: center;
+        width: 96.25%;
+        margin: 0 auto;
+    }
+}
+
+@media (max-width:500px) {
+    .container {
+        width: 100%;
+    }
+   
+
+    .main-head nav {
+        display: flex;
+        flex-flow: column wrap;
+        justify-content: center;
+        width: 100%;
+        border-top: 1px solid grey;
+    }
+
+    .main-head nav a {
+        display: flex;
+        font-size: 2.2rem;
+        color: lightgrey;
+        border-bottom: 1px solid grey;
+        width: 100%;
+        justify-content: center;
+        height: 70px;
+        align-items: center;
+    }
+    @keyframes example {
```

Comment:
This is my favorite part of your whole project 😍 I love the animations you add in and all of these shades look really fade in and out perfectly!